cmd/example-posix/main.go

```diff
@@ -100,27 +100,16 @@ func main() {
 		return posix.WriteCheckpoint(*storageDir, n)
 	}
 	if *initialise {
+		if err := os.RemoveAll(*storageDir); err != nil {
```

Comment:
This seems bold :)
I guess it's just an example, but probably don't want people to copy this behaviour!

Reply:
Yeah I've got mixed feelings about it. I can make it fail instead if there are contents in the directory?

Reply:
Yeah, seems safer